### PR TITLE
Feat:Create blog section flutter

### PR DESCRIPTION
**Pull request by cp-ashish-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [ ] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [ ] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

- Create Blog Section for flutter development Page So user can get detail idea about the blogs written by the canopas on flutter
- Image for the mobile UI Is as below 
image

- video for mobile UI
[Screencast from 21-11-23 04:31:42 PM IST.webm](https://github.com/canopas/canopas-website/assets/140039768/95ceb103-1ff3-4e18-9415-093a8b669894)

- Image for the desktop UI is as below
image

- The LightHouse score for mobile UI Is as below
Screenshot from 2023-11-21 16-19-59

- The LightHouse score for the desktop UI is as below
image

### CODE REVIEW

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        >
+          <img
+            :src="blog.image[2]"
+            :srcset="`${blog.image[2]} 800w`"
+            :alt="blog.title"
+            class="object-fill h-full w-full cursor-pointer rounded-2xl"
+            loading="lazy"
+          />
+          <div
+            class="ease absolute top-1/2 left-1/2 right-1/2 rounded-2xl transition-all duration-700 opacity-0 overflow-hidden w-0 h-0 hover:opacity-100 group-hover:h-full group-hover:w-full group-hover:top-0 group-hover:left-0 group-hover:right-0"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:61]: here also border radius is not same as figma

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+          <swiper-slide v-for="(blog, index) in blogs" :key="index">
+            <div
+              class="scale-75 text-left transition-all duration-200 ease-out"
+              @click="openBlog(blog.link, 'tap_flutter_app_blog_section')"
+            >
+              <img
+                :src="blog.image[0]"
+                :srcset="`${blog.image[0]} 400w, ${blog.image[1]} 800w`"
+                :alt="blog.title"
+                class="h-full w-full rounded-lg object-cover"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:22]: Use the same border radius as figma

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+          />
+          <div
+            class="ease absolute top-1/2 left-1/2 right-1/2 rounded-2xl transition-all duration-700 opacity-0 overflow-hidden w-0 h-0 hover:opacity-100 group-hover:h-full group-hover:w-full group-hover:top-0 group-hover:left-0 group-hover:right-0"
+            :class="blog.bgColor"
+          >
+            <div
+              class="absolute w-full -top-72 left-1/2 text-center -translate-x-1/2 transition-all duration-1000 transform group-hover:top-1/2"
+            >
+              <span
+                v-html="blog.hovertitle"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:68]: title is not in center for all cards

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  },
+  components: {
+    Swiper,
+    SwiperSlide,
+    AspectRatio,
+  },
+  inject: ["mixpanel"],
+};
+</script>
+<style lang="postcss">
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:224]: use this css as scoped css check in other pages its affecting there

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+          <img
+            :src="blog.image[1]"
+            :srcset="`${blog.image[1]} 800w`"
+            class="object-fit h-full w-full cursor-pointer rounded-2xl"
+            loading="lazy"
+            :alt="blog.title"
+          />
+          <div
+            class="ease absolute top-1/2 left-1/2 right-1/2 rounded-2xl transition-all duration-700 opacity-0 overflow-hidden w-0 h-0 hover:opacity-100 group-hover:h-full group-hover:w-full group-hover:top-0 group-hover:left-0 group-hover:right-0"
+            :class="blog.bgColor"
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:121]: I think all the backgrounds are the same. so, you can use it directly

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+        >
+          <img
+            :src="blog.image[2]"
+            :srcset="`${blog.image[2]} 800w`"
+            :alt="blog.title"
+            class="object-fill h-full w-full cursor-pointer rounded-xl"
+            loading="lazy"
+          />
+          <div
+            class="bg-[#121212] ease absolute top-1/2 left-1/2 right-1/2 rounded-xl transition-all duration-700 opacity-0 overflow-hidden w-0 h-0 hover:opacity-100 group-hover:h-full group-hover:w-full group-hover:top-0 group-hover:left-0 group-hover:right-0"
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:61]: Felt a bit slow hover effect compared to Figma

File: nuxt-frontend/components/flutter-app-development/BlogSection.vue [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+    <div class="container hidden lg:block lg:min-h-[580px]">
+      <p class="mb-14 text-center header-2 text-black-core/[0.87]">Our blogs</p>
+      <div class="flex flex-row space-x-5">
+        <div
+          @click="openBlog(blog.link, 'tap_flutter_app_blog_section')"
+          v-for="blog in blogs.slice(0, 1)"
+          :key="blog"
+          class="content group relative flex basis-[65%]"
+        >
+          <img
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/flutter-app-development/BlogSection.vue:53]: The very first image looks stretched in height can you please have a look?

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-dharti-r — APPROVED** [review]

**Pull request merged**